app/Main.hs

```diff
@@ -75,11 +77,21 @@ showFormat Hadolint.GitlabCodeclimateJson = "gitlab_codeclimate"
 showFormat Hadolint.Checkstyle = "checkstyle"
 showFormat Hadolint.Codacy = "codacy"
 
+toNofailSeverity :: String -> Maybe Rule.DLSeverity
+toNofailSeverity "error" = Just Rule.DLErrorC
+toNofailSeverity "warning" = Just Rule.DLWarningC
+toNofailSeverity "info" = Just Rule.DLInfoC
+toNofailSeverity "style" = Just Rule.DLStyleC
+toNofailSeverity "ignore" = Just Rule.DLIgnoreC
+toNofailSeverity "none" = Just Rule.DLIgnoreC
```

Comment:
Why do we need the none value?

Reply:
While `ignore` fits well within the context of the severity, I think it is terrible user experience and very unintuitive. If you don't share this opinion, I'll remove it and add documentation accordingly.

Reply:
makes sense to me, you can leave it